views/migrate-live-services.erb

```diff
@@ -0,0 +1,47 @@
+
+<% content_for :head do %>
+	<title>Migrate a live service to GOV.UK PaaS - <%= settings.title %></title>
+<% end %>
+
+<% content_for :breadcrumb do %>
+	<%== erb :'partials/_breadcrumb', :locals => {name: 'Get started', href: request.path_info, active: true} %>
```

Comment:
I don't really understand how this works or whether it needs to be modified for this page.